style/react/README.md

```diff
@@ -0,0 +1,50 @@
+# React
```

Comment:
I think maybe this should be in best-practices instead (it's not about formatting) but the different between those is fuzzy. I wonder if they should be merged at this point.

Reply:
I agree, almost everything that came to mind for React was a best=practices thing (since style is already covered by the TypeScript guide).

I would be in favor of merging style and best practices guides. It might be worth calling out style guidelines vs best practices where both exist, but it also allows for guidelines which might be in more of a gray area.

Reply:
I have opened an issue to discuss combining them: https://github.com/thoughtbot/guides/issues/610